### ISSUE TITLE
Play nice with others

### ISSUE THREAD

**Issue by alexellis** [plunder-app/plndr-cloud-provider#24]
I could see the Plndr LoadBalancer being useful with inlets-operator.

There may also be some combination with MetalLB or another LB project.

LBs don't usually "Play well with others" - MetaLB has several outstanding issues and feature requests for an ignore mode or annotation. The inlets-operator has this already -> https://github.com/inlets/inlets-operator#annotations-and-ignoring-services

Would you consider adding a similar annotation so that we can use various tools together?

**Comment by thebsdbox** [comment]
Yeah, I'm hoping to start ramping up a lot of work on this soon.. we can easily have the controller lookup annotations to stop it stealing services :-D
